microsoft-edge/extensions-chromium/publish/api/using-addons-api.md

```diff
@@ -33,35 +91,79 @@ To submit suggestions and feedback, enter an [Issue about the Add-ons API](https
 
 
 <!-- ====================================================================== -->
-## Before you begin
+## Enabling the Update REST API at Partner Center
+
+To use the Microsoft Edge Add-ons API, you first need to enable the API for your project in the Microsoft Partner Center by creating API credentials, as follows:
+
+##### [v1.1](#tab/v1-1)
+
+1. At your [Partner Center developer dashboard](https://partner.microsoft.com/dashboard/microsoftedge/public/login?ref=dd), sign in to the account that you used to publish an add-on.
+
+1. Under the **Microsoft Edge** program, select **Publish API**.
+
+1. On the **Publish API** page, click the **Create API credentials** button.  This step may take a few minutes to finish.
 
-To use the Microsoft Edge Add-ons API, you need to enable the API for your project in the Microsoft Partner Center, by creating API credentials. Use the following steps to create API credentials.
+   The API credentials have now been created; you've enabled or renewed the API.  The following items are now displayed on the **Publish API** page:
+   * **Client ID**
+   * **API key**
+   * **Expiry date**
 
-1. Visit Microsoft Partner Center and sign in to the account that you used to publish an add-on.
+   ![The 'Publish API' page at Partner Center after clicking 'Create API credentials', now showing Client ID and API key](./using-addons-api-images/create-api-credentials-button.png)<!-- todo: add/use new png showing v1.1.  What's the title of the page that contains the UI control to select v1 vs v1.1?  What's the label of the UI control?  Is the UI control an Option button, or a Button? -->
+
+1. Write down the following:
+   * The **Client ID**.
+   * The **API key**
+
+   You'll use these values in a later step.<!-- todo: why write down these values?  specify which later step uses the value, maybe link -->
```

Comment:
Why write down the ClientID & API key?
State what will be done w/ ClientID and API key, which step(s)?

Reply:
removed hidden todo comment, rewrote sufficiently for review.  eg:

1. Write down the ClientID and API key.  When you use a REST endpoint, you'll specify these in the request header.

The ClientID and API key are automatically generated.  You'll use these values as request headers when using a REST endpoint:

   ``REST
   `Authorization: ApiKey <api key>`
   `X-ClientID: <client ID>`
   ```